plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergDynamicPartitionPruningTest.java

```diff
@@ -35,7 +36,8 @@ protected QueryRunner createQueryRunner()
         return IcebergQueryRunner.createIcebergQueryRunner(
                 EXTRA_PROPERTIES,
                 ImmutableMap.of("iceberg.dynamic-filtering.wait-timeout", "1h"),
-                REQUIRED_TABLES);
+                Optional.empty(),
+                SchemaInitializer.builder().withClonedTpchTables(REQUIRED_TABLES).build());
```

Comment:
This is a typical case.
Would it work, if we passed `Iterable<TpchTable<?>> tables` here and  schema initialized was created for us?